common/app/services/RedirectService.scala

```diff
@@ -41,16 +40,19 @@ object RedirectService {
   }
 
   implicit val destinationFormat: AnyRef with DynamoFormat[Destination] =
-    DynamoFormat.xmap[Destination, Map[String, String]] {
-      // map -> destination (i.e. reads)
-      case m if m.contains("destination") => Right(PermanentRedirect(m("source"), m("destination")))
-      case m if m.contains("archive")     => Right(ArchiveRedirect(m("source"), m("archive")))
-      case _                              => Left(MissingProperty)
-    } {
-      // destination -> map (i.e. writes)
-      case PermanentRedirect(source, destination) => Map("source" -> source, "destination" -> destination)
-      case ArchiveRedirect(source, archive)       => Map("source" -> source, "archive" -> archive)
-    }
+    DynamoFormat.xmap[Destination, Map[String, String]](
```

Comment:
Gets rid of error:

```
[error] /Users/Ioanna_Kokkini/code/frontend/common/app/services/RedirectService.scala:44:7: not enough arguments for method xmap: (r: Map[String,String] => Either[org.scanamo.DynamoReadError,services.RedirectService.Destination], w: services.RedirectService.Destination => Map[String,String])(implicit f: org.scanamo.DynamoFormat[Map[String,String]]): org.scanamo.DynamoFormat[services.RedirectService.Destination].
[error] Unspecified value parameter w.
[error]       {
[error]       ^
```

Related to [this commit](https://github.com/scanamo/scanamo/pull/588/files#diff-7104c8da3ec1ec582e990f098460dbb8172dd095bf60dc7c2912e1b136a0b86dR190) which changed the `xmap` signature.